src/test/java/net/minecraftforge/debug/CanApplyAtEnchantingTableTest.java

```diff
@@ -8,10 +8,10 @@
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
 import net.minecraftforge.fml.common.registry.GameRegistry;
 
-@Mod(modid="canapplyatenchantingtabletest", name="CanApplyAtEnchantingTableTest", version="0.0.0")
+@Mod(modid = "canapplyatenchantingtabletest", name = "CanApplyAtEnchantingTableTest", version = "0.0.0")
 public class CanApplyAtEnchantingTableTest
 {
-    public static final boolean ENABLE = true;
+    public static final boolean ENABLE = false;
 
     public static final Item testItem = new Item()
```

Comment:
This needs `isItemTool` renaming to `isEnchantable`, and the missing `@Override` added.